src/physics/sitetype.jl

```diff
@@ -220,20 +233,67 @@ function state(sset::Vector{<:Index},
   return state(sset[j],st)
 end
 
-function siteinds(d::Integer,
-                  N::Integer; kwargs...)
-  return [Index(d,"Site,n=$n") for n=1:N]
+#---------------------------------------
+#
+# siteind system
+#
+#---------------------------------------
+
+space(st::SiteType; kwargs...) = throw(MethodError("Overload of \"space\",\"siteind\", or \"siteinds\" functions not found for Index tag: $(tag(st))"))
+
+function siteind(st::SiteType; add_tags="", kwargs...) 
```

Comment:
Could we just call it `addtags`, since that is the name of the function?

Reply:
Changed in latest commits